build/DependencyVersions.props

```diff
@@ -15,7 +15,7 @@
     <TemplateEngineTemplateVersion>1.0.0-beta2-20170410-189</TemplateEngineTemplateVersion>
     <TemplateEngineTemplate2_0Version>1.0.0-beta2-20170410-189</TemplateEngineTemplate2_0Version>
     <PlatformAbstractionsVersion>2.0.0-preview1-001960</PlatformAbstractionsVersion>

```

Comment:
I think it would be nice to keep `PlatformAbstractionsVersion` and `DependencyModelVersion` in sync.

Maybe have a separate property `$(CoreSetupPrereleaseVersion)`.  And then we could update it in our update-dependencies script.

Then all 3 of these properties would be kept in sync:

`<CLI_SharedFrameworkVersion>2.0.0-preview1-002021-00</CLI_SharedFrameworkVersion>`
`<PlatformAbstractionsVersion>2.0.0-preview1-001960</PlatformAbstractionsVersion>`
`<DependencyModelVersion>2.0.0-preview1-002022</DependencyModelVersion>`

@livarcocc - is this something your team could pick up?